### PR TITLE
Specify all node packages and versions for upgrade

### PR DESCRIPTION
**Pull request by mtnbikenc**
Package version is required for all dependent node packages or yum will install the latest version

### CODE REVIEW

**Comment by mtnbikenc** [comment]
This is a forward port of #8446

File: roles/openshift_node/tasks/upgrade/rpm_upgrade.yml [hunk 1]
```diff
@@ -14,6 +21,7 @@
     - result.stdout is search(".*No package .* available.*") or result is failed
   vars:
     openshift_node_upgrade_rpm_list:
+      - "{{ openshift_service_type }}{{ openshift_pkg_version | default('') }}"
```
Comment (michaelgugino) [inline roles/openshift_node/tasks/upgrade/rpm_upgrade.yml]: openshift_pkg_version should always be defined, let's clean up these unnecessary defaults.

Comment (mtnbikenc) [reply roles/openshift_node/tasks/upgrade/rpm_upgrade.yml]: Ok

**Comment by mtnbikenc** [comment]
Updated to remove unnecessary defaults.

**Pull request merged**